HHS/Head-Start-TTADP
# [PROD] Add Creator Role to AR, Update Display and CSV for Creator Role, save sort + filters to local storage

## Description of change

**TTAHUB-372:** Added creator role to AR. If creator has one role this will be automatically selected. If the user has more than one role they will be required to select the role they created the AR under.

**TTAHUB-693:** Update Alert, Approved AR, and CSV to reflect new creator role.

**TTAHUB-252:** Users filtering, sorting, and pagination is preserved through navigation across the site and the back button

## How to test

**TTAHUB-372:** Create a AR as a user with a single role. Create a AR as a user with multiple roles. 

**TTAHUB-693:** The Alerts, Approved AR, and CSV should now show the creator role else all roles for the user.
NOTE: If a report is in draft and the user has not selected a creator role. The creator field on the alerts table with only show the creator name.

**TTAHUB-252:** On the landing page or the dashboard, sort and change pages on the approved activity reports table. Navigate to a different page, then return. Your position is saved on the table. Change filters, rinse, and repeat. Your position is still saved.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-252
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-372
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-693

## Checklists

### Every PR


~- [ ] API Documentation updated~
~- [ ] Boundary diagram updated~
~- [ ] Logical Data Model updated~
~- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions~

### Production Deploy


### After merge/deploy
